### PR TITLE
feat: helm chart

### PR DESCRIPTION
**Pull request by matthieu-foucault**

### CODE REVIEW

File: helm/cas-ggircs/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: cas-ggircs
+description: Helm chart for the GGIRCS database and ETL
+type: application
+version: 0.1.0
+appVersion: 0.1.0
+dependencies:
+- name: cas-postgres
+  version: "0.2.0"
+  repository: https://bcgov.github.io/cas-postgres/
```
Comment (wenzowski) [inline helm/cas-ggircs/Chart.yaml:10]: woot woot! the `cas-postgres` chart will need to have all its dependencies published (no file dependencies) for this to work

File: helm/cas-ggircs/values.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+image:
+  etl:
+    repository: docker-registry.default.svc:5000/wksv3k-tools/cas-ggircs-etl
```
Comment (wenzowski) [inline helm/cas-ggircs/values.yaml:3]: should we consider dumping this to gcr eventually as well? (not right now)

File: shipit.yml [hunk 3]
```diff
@@ -1,5 +1,7 @@
 deploy:
   override:
     - git submodule update --init --quiet
-    - make install:
-        timeout: 3600
+    - helm dep up ./helm/cas-ggircs
+    - helm upgrade --install --atomic --timeout 900s --namespace "$ENVIRONMENT" --set image.etl.tag "$REVISION" cas-ggircs ./helm/cas-ggircs:
+        timeout: 900
+    - curl -u ${AIRFLOW_USERNAME}:${AIRFLOW_PASSWORD} -X POST https://cas-airflow-${ENVIRONMENT}.pathfinder.gov.bc.ca/api/experimental/dags/ggircs_deploy_db/dag_runs
```
Comment (wenzowski) [inline shipit.yml:7]: any reason to patch this here rather than in `make install`?

File: helm/cas-ggircs/Chart.yaml (hunk 1 above)
Comment (wenzowski) [reply helm/cas-ggircs/Chart.yaml:10]: see https://github.com/bcgov/cas-postgres/pull/50/files#r436111322

Comment (matthieu-foucault) [reply helm/cas-ggircs/Chart.yaml:10]: It's been published to the ghpages, and I'm pretty sure this works as-is, with the file dependency

Comment (wenzowski) [reply helm/cas-ggircs/Chart.yaml:10]: I'll have to look into why that works now; didn't used to

File: shipit.yml (hunk 3 above)
Comment (matthieu-foucault) [reply shipit.yml:7]: Nope I can move this to the make install command.

I mainly wanted to check that it was working on shipit. I manually redeployed shipit and added those AIRFLOW_USERNAME and AIRFLOW_PASSWORD env variables to the pod, but I am not completely sure if the shipit jobs can see those env variables (I would assume so)

**Pull request merged**